crates/astria-bridge-contracts/src/lib.rs

```diff
@@ -467,54 +495,97 @@ where
 
 #[derive(Debug, thiserror::Error)]
 #[error(transparent)]
-pub struct GetWithdrawalActionsError(GetWithdrawalActionsErrorKind);
+pub struct GetWithdrawalLogsError(GetWithdrawalLogsErrorKind);
```

Comment:
I think you are leaking concepts that don't appear at the high level. Please change this back.